templates/crd-proxydefaults.yaml

```
@@ -86,6 +86,13 @@ spec:
                     description: Mode is the mode that should be used for the upstream connection. One of none, local, or remote.
                     type: string
                 type: object
+              transparentProxy:
+                description: TransparentProxy controls configuration specific to proxies in transparent mode.
```

```suggestion
                description: TransparentProxy controls configuration specific to proxies in transparent proxy mode.
```
Thoughts? Do we use "transparent mode" anywhere?

---

Oh I guess it's proxies in transparent mode, 🤷🏽 ?

---

Exactly. It is about the proxies being in transparent mode 😅